Code/Framework/AzCore/Tests/AZStd/Pair.cpp

```diff
@@ -17,17 +17,17 @@ namespace UnitTest
     // Fixture for non-typed tests
     template<typename TestConfig>
     class CompressedPairTest
-        : public AllocatorsFixture
+        : public LeakDetectionFixture
     {
     protected:
         void SetUp() override
         {
-            AllocatorsFixture::SetUp();
+            LeakDetectionFixture::SetUp();
         }
 
         void TearDown() override
         {
-            AllocatorsFixture::TearDown();
+            LeakDetectionFixture::TearDown();
         }
     };
```

Comment:
Instances of fixtures like this where the derived class doesn't do anything unique, can we simply omit these overrides of SetUp and TearDown?

Reply:
In general, there seems to be a lot of variance in how these fixture subclasses are written.  Some explicitly call SetUp and TearDown, like this one, others only call SetUp.  Some don't call SetUp/TearDown of the base class at all in their overrides.  I get that this is mostly a find-and-replace change, but if there are opportunities to remove unnecessary lines then I'm in favor.

Reply:
I used
```
grep -Pzo '(?s)void SetUp\(\) override\n *{ *\n *LeakDetectionFixture::SetUp\(\);\n *}\n' Code Gems -rIl
```

to find some 13 files that had empty `SetUp()` functions, and cleaned those up.

The `LeakDetectionFixture` does its work on construct/destruct, so there's no getting around those calls.

Reply:
But the base class SetUp and TearDown functions will be called if the derived class implementations don't exist